### PR TITLE
Use gin for server

### PR DESCRIPTION
**Pull request by roger-dodger**
Use Gin Web Framework for HTTP routing in server part. This does not bring any advantages now, but will help us in a later stage when we add authentication for client-server communication.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -125,27 +132,31 @@ func (s *SpectreServer) renderHandler(w http.ResponseWriter, r *http.Request) {
 			AddGrid: addGrid,
 		},
 		Filter: &extraction.FilterOptions{
-			SDR:        sdr,
-			Identifier: identifier,
+			SDR:        parsedQueryParameters.Sdr,
```
Comment (finfinack) [inline server/server.go:135]: Can you capitalize to .SDR please?

File: export/spectre.go [hunk 2]
```diff
@@ -47,6 +54,15 @@ func (s *SpectreServer) Write(ctx context.Context, samples <-chan sdr.Sample) er
...
 			continue
 		}
+		respBody, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			glog.Warningf("error reading POST body: %s\n", err)
+		}
+
+		collectResponseBody := collectResponse{}
+		json.Unmarshal(respBody, &collectResponseBody)
+		glog.Infof("submitted %v samples to server %s", collectResponseBody.SampleCount, s.Server)
```
Comment (finfinack) [inline export/spectre.go:64]: %d please

**Review by finfinack — CHANGES_REQUESTED** [review]

File: server/server.go (hunk 1 above)
Comment (roger-dodger) [reply server/server.go]: done

File: export/spectre.go (hunk 2 above)
Comment (roger-dodger) [reply export/spectre.go]: done

**Review by finfinack — APPROVED** [review]
Thank you! Now some Gin and Tonic...

**Comment by roger-dodger** [comment]
Thank you very much

**Pull request merged**